R/parameters_groupspecific.R

```diff
@@ -74,15 +77,19 @@ parameters_groupspecific <- function(model, indices = c("Coefficient", "SE"), ..
                                   sep = "_")
 
     # If nested, separate groups
-    if(grepl(":", random_group)) {
+    if (grepl(":", random_group)) {
       groups <- as.data.frame(t(sapply(strsplit(wide[[random_group]], ":"), function(x) as.data.frame(t(x)))))
       names(groups) <- unlist(strsplit(random_group, ":"))
       wide <- cbind(groups, wide)
       wide[random_group] <- NULL
       random_group <- names(groups)
     }
 
+    # merge and sort
+    df_random[["__sort_id"]] <- 1:nrow(df_random)
```

Comment:
we really should add a `data_merge()` function in datawiz (@IndrajeetPatil) to streamline this kind of stuff